src/transformers/models/speech_to_text_2/modeling_speech_to_text_2.py

```diff
@@ -870,7 +870,7 @@ def forward(
         >>> from datasets import load_dataset
 
         >>> feature_extractor = Wav2Vec2FeatureExtractor()
-        >>> tokenizer = Speech2Text2Tokenizer.from_pretrained(_CHECKPOINT_FOR_DOC)
+        >>> tokenizer = Speech2Text2Tokenizer.from_pretrained({_CHECKPOINT_FOR_DOC})
```

Comment:
We also need to change `r"""` to `rf"""` in line 792 (sorry can't comment there)

Reply:
Done! (Looks like this is the first occurrence of `rf"""` in the library)

Reply:
Looks like `rf"""` causes problem 🤔.

Reply:
Could you try just `f"""` in this case?

Reply:
Otherwise ok for me to just revert to your very first version (just copy-paste the checkpoint name), if it doesn't work or to use `format(...)` instead

Reply:
`f"""` also causes problems. I reverted back to the first version, and will merge with it.

(we can see how to use the format string for docstring later.)